includes/class-wc-payment-token-swedbank-pay.php

```diff
@@ -36,7 +34,7 @@ class WC_Payment_Token_Swedbank_Pay extends WC_Payment_Token_CC {
 	public function get_display_name( $deprecated = '' ) {
 		ob_start();
 		?>
-		<img src="<?php echo WC_HTTPS::force_https_url( WC()->plugin_url() . '/assets/images/icons/credit-cards/' . $this->get_card_type() . '.png' ) ?>"
+		<img src="<?php echo WC_HTTPS::force_https_url( WC()->plugin_url() . '/assets/images/icons/credit-cards/' . $this->get_card_type() . '.png' ); ?>"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class 'WC_HTTPS' in method 'get_display_name'.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-payments/pullRequest?prid=5340032)